src/main/resources/assets/js/models/Collection.js

```diff
@@ -0,0 +1,27 @@
+var Backbone = require("backbone");
+
+module.exports = Backbone.Collection.extend({
```

Comment:
In general this should ensure that you kill of any ongoing polls before you start the new one:
https://github.com/mesosphere/marathon/pull/1066/files#diff-6876cc8f56c80c7276969fd16e24b309R400